community/kernel/src/test/java/org/neo4j/kernel/impl/store/countStore/DiffMapValuesTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.store.countStore;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.neo4j.kernel.impl.store.counts.keys.CountsKey;
+
+import static org.neo4j.kernel.impl.store.counts.keys.CountsKeyFactory.nodeKey;
+
+/**
+ * A test for the test class generating all intermediate states the count
+ */
+public class DiffMapValuesTest
```

Comment:
I don't think we should testing test code. Let's simply move the static methods where they are used.